tests/py-flux-test.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+from flux.thermal import PccThermalModel1D
+from flux.grids import setgrid
+
+class PyFluxTest(unittest.TestCase):
+    def test_conductionQ(self):
+
+        # import template output
+        df_template = pd.read_csv("/home/sberton2/Works/NASA/radiosity/python-flux/tests/template.out",sep='\s+').iloc[:,:-1]
+        df_template.columns = ["depth","temperature","heat_flux"]
+
+        # generate 1D grid
+        nz = 60
+        zfac = 1.05
+        zmax = 2.5
+        z = setgrid(nz=nz, zfac=zfac, zmax=zmax)
+        # print(z)
+
+        # set-up thermal model
+        model = PccThermalModel1D(nfaces=1,z=z,T0=210.,ti=120.,rhoc=960000.,emissivity=1.,Fgeotherm=0.2)
+
+        # simulated Q input at t steps (dt=495661.76666666666 s)
+        Qsim = [np.array([470.84688077122127]), np.array([470.20160144930207]),
+                np.array([468.26753214941181]), np.array([465.04997402136058])]
```

Comment:
Change to Qsim = np.array([470.846..., blah, blah]) or just Qsim = [470.846..., ...]

Reply:
I actually did this because `model.step` requires `Qsim = np.array([])` (i.e., an np.array of len>0) ... moved the recasting to the function.

Reply:
and/or maybe `thermal.step` should be made more flexible ...

Reply:
Hmmm, OK. I'll try to remember to look into this, but want to fix #6 first (working on this now...). If you can make an issue for improving `thermal.step`, that'd be appreciated, and will try to get to it soon.